components/nav/NamespaceFilter.vue

```diff
@@ -76,7 +79,13 @@ export default {
       const namespaces = sortBy(
         this.$store.getters[`${ inStore }/all`](NAMESPACE),
         ['nameDisplay']
-      ).filter( N => this.isVirtualCluster ? !N.isSystem : true);
+      ).filter( (N) => {
+        const isFleets = get(N, `metadata.labels."${ FLEET.MANAGED }"`) === 'true';
```

Comment:
We use this in 2 places - could we put this on the model for the namespace resource?

Reply:
Fixed